en/access-control.texy

```diff
@@ -77,11 +77,12 @@ If the login credentials are not valid, authenticator throws an [api:Nette\Secur
 try {
 	// we try to log the user in
 	$user->login($username, $password);
+
 	// ... and redirect upon success
 	$this->redirect(...);
 
 } catch (Nette\Security\AuthenticationException $e) {
-	echo 'Login error: ', $e->getMessage();
+	$this->flashMessage('Login error: ' . $e->getMessage(), 'warning');
```

Comment:
`$this->flashMessage('The username or password you entered is incorrect.', 'warning');` seems better